### PR TITLE
Axios instance

### PR DESCRIPTION
**Pull request by Mazoda**

### CODE REVIEW

File: src/Components/InputWithIcon/SearchInputWithIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    return (
+
+        <Search>
+            <SearchIconWrapper>
+                <SearchIcon/>
+            </SearchIconWrapper>
+            <StyledInputBase
+                placeholder="Search…"
+                inputProps={{'aria-label': 'search'}}
+            />
```
Comment (Halasalhab) [inline src/Components/InputWithIcon/SearchInputWithIcon.jsx]: The placeholder should come for the container component as props which in this case it's the Home, as this is only a representational component

File: src/Components/InputWithIcon/SearchInputWithIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            transition: theme.transitions.create('width'),
+            width: '100%',
+            [theme.breakpoints.up('sm')]: {
+                width: '12ch',
+                '&:focus': {
+                    width: '20ch',
+                },
+            },
+        },
+    }));
```
Comment (Halasalhab) [inline src/Components/InputWithIcon/SearchInputWithIcon.jsx]: Can you please separate these into different files? as it's easier to track an edit in the future.

File: src/Services/network.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    baseURL: ""
+});
+
+
+export const fetchData = async (url) => {
+    try {
+        const response = await client.get(url);
+        return response.data;
+    } catch (error) {
+        return {}
```
Comment (Halasalhab) [inline src/Services/network.js]: In case of an error, it should return the error message based on the backend response, not {}

**Review by Halasalhab — CHANGES_REQUESTED** [review]
Please review the comments

File: src/Components/InputWithIcon/SearchInputWithIcon.jsx (hunk 1 above)
Comment (Mazoda) [reply src/Components/InputWithIcon/SearchInputWithIcon.jsx]: I am not sure why the Compomnents folder still exists in the branch even though i deleted it.
 I pushed it by mistake.
 I will make another branch for it.

File: src/Components/InputWithIcon/SearchInputWithIcon.jsx (hunk 2 above)
Comment (Mazoda) [reply src/Components/InputWithIcon/SearchInputWithIcon.jsx]: I already deleted it in the [https://github.com/JSD-0423/frontend-final-3/pull/1/commits/6c24fcd442d01de925671d82e8e48812ba609529](url)

**Comment by Mazoda** [comment]
Changes made to mentioned problems @Halasalhab

**Review by Halasalhab — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]

**Pull request merged**